spec/gameSpec.js

```diff
@@ -0,0 +1,122 @@
+var Game = require('../src/game.js');
+var TenthFrame = require('../src/tenthFrame.js');
+var Frame = require('../src/Frame.js');
+
+
+describe("Game", function() {
+
+    beforeEach(function() {
+      game = new Game();
+    });
+
+    it("is an instance of Game", function() {
+      expect(game).toEqual(jasmine.any(Game));
+    });
+
+    it("is initialised with ten empty frames", function() {
+      expect(game.frames.length).toEqual(10);
+    });
+
+    it("its last frame should be of type 'tenthFrame'", function() {
+      expect(game.frames.pop()).toEqual(jasmine.any(TenthFrame));
+    });
+
+    it("can return a given frame", function(){
+      expect(game.getFrame(9)).toEqual(jasmine.any(TenthFrame));
+    });
+
+    it("can correctly update frames with an input score method", function(){
+      game.inputScore(10); // 1
```

Comment:
You should probably have a helper function here that does something like:
```js
function addMultipleRolls(rolls) {
    for (var i=0; i < rolls.length; i++) {
        game.inputScore(rolls[i]);
    }
}
```

Reply:
Yep, I had used a few helper functions similar to this elsewhere - I just didn't get around to refactoring it. I think sometimes it is good to 'get it working' and then refactor (which backfires when you run out of time :-(.